phys2bids/bids.py

```diff
@@ -1,6 +1,8 @@
+from csv import reader
 import os
 import logging
 
+import yaml
```

Comment:
I _think_ that we decided all the "from x import y" to go at the end of their relative section (so "from csv import reader" should go before pathlib but after logging).
yaml is a non-core, so it goes after pathlib.